### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by raeich**
replacing veggies with blueberries

### CODE REVIEW

**Comment by raeich** [comment]
@mjfaade let me know what you think!

File: README.md [hunk 1]
```diff
@@ -5,5 +5,5 @@ Hello world! Take a look at my top 5 favorite foods :)
 1. Donuts
```
Comment (mjfaade10) [inline README.md:5]: Can you be more specific about what type of donut you like?

File: README.md [hunk 2]
```diff
@@ -5,5 +5,5 @@ Hello world! Take a look at my top 5 favorite foods :)
...
 2. Tacos
 3. Sushi
```
Comment (mjfaade10) [inline README.md:7]: I love sushi! What is your favorite roll?

File: README.md [hunk 3]
```diff
@@ -5,5 +5,5 @@ Hello world! Take a look at my top 5 favorite foods :)
...
-4. All the veggies
+4. Blueberries
 5. Chocolate
```
Comment (mjfaade10) [inline README.md:9]: We want to know the details! What types of chocolate?

**Review by mjfaade10 — DISMISSED** [review]
Thanks for sharing! Here's my feedback :)

File: README.md (hunk 2 above)
Comment (raeich) [reply README.md:7]: Spicy tuna 😁👌

**Comment by raeich** [comment]
@mjfaade10 updated the list based on your feedback. let me know if this is good to go.

File: README.md [hunk 4]
```diff
@@ -2,8 +2,8 @@
 
 Hello world! Take a look at my top 5 favorite foods :)
 
-1. Donuts
+1. Chocolate donuts with sprinkles
```
Comment (mjfaade10) [inline README.md:5]: I love sprinkles!

File: README.md [hunk 5]
```diff
@@ -2,8 +2,8 @@
...
 2. Tacos
 3. Sushi
-4. All the veggies
-5. Chocolate
+4. Blueberries
+5. Dark chocolate w/caramel
```
Comment (mjfaade10) [inline README.md:9]: Thanks for sharing!

File: README.md [hunk 6]
```diff
@@ -2,8 +2,8 @@
 
 Hello world! Take a look at my top 5 favorite foods :)
 
-1. Donuts
+1. Chocolate donuts with sprinkles
 2. Tacos
 3. Sushi
-4. All the veggies
-5. Chocolate
+4. Blueberries
```
Comment (mjfaade10) [inline README.md:8]: team strawberries :)

**Pull request merged**